### PR TITLE
Code changes to handle error messages in NW provider

### PR DESCRIPTION
**Pull request by hsridharan**
## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [x] post a screenshot of templates and/or gallery changes
* [x] ensure all steps have meaningful names
* [x] ensure all parameters and grid columns have display names set so they can be localized
* [x] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [x] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [x] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [x] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

Code changes to add error messages for the following scenarios :
1. When the Provider is not added, and the user tries to access the workbook
CreateProvider
2. When none of the NW providers are successfully created.  
ProviderNotcreated
3. When the Log Analytics tables are created but the data is not yet ingested.

### CODE REVIEW

**Comment by PakDLiu** [comment]
For the newer version of SapMonitor, how did you determine if there is a new version?
Also, the error message should probably say "...deleting and creating a new SAP monitor"
And if this were to happen, will it still show the current workbook and metrics?

**Comment by hsridharan** [comment]
@PakDLiu - I added SAPMON_VERSION column to the LA tables. Here in the WB, I am checking if the version is < version in LA table then I am showing the alert in the WB. I updated the error message as u recommended.

**Comment by PakDLiu** [comment]
I am not sure if we should have a validation on the payload version.
We release new versions all the time (bug fixes, new providers) that may not have any affect on the netweaver provider.

**Comment by hsridharan** [comment]
@PakDLiu - Yes, Ideally this message would never appear because most of the new AMS will have version 2.8 or more and quey will always pass. We'll add the queries to check the newly added column or anything related in the future when the need comes into the picture.

**Comment by persiaAziz** [comment]
I don't think we can use the term "SAP Monitor" as the third warning above for legal reasons.

**Review by persiaAziz — CHANGES_REQUESTED** [review]
Please make sure to not use term as "SAP Monitor" as it is not legally allowed

File: Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook [hunk 1]
```diff
@@ -8,6 +8,143 @@
...
+              "parameterName": "param_check_error",
+              "comparison": "isEqualTo",
+              "value": "2"
+            },
+            "name": "IngestionError"
+          },
+          {
+            "type": 1,
+            "content": {
+              "json": "We have a newer version of SAP monitor. If you would like to see more metrics, please upgrade by deleting and creating a new SAP monitor.",
```
Comment (persiaAziz) [inline Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook:134]: Why would they need to delete and deploy completely new monitor? We upgrade them automatically

**Review by persiaAziz — APPROVED** [review]

**Review by PakDLiu — APPROVED** [review]

**Pull request merged**